README.md

```diff
@@ -103,6 +101,24 @@ pip install -r dev-requirements.txt
 pytest tests/
 ```
 
+[Git LFS](https://git-lfs.com/) must be installed to get all the files. This is packaged as `git-lfs` on Ubuntu, Fedora or macOS Homebrew. For Equinor RGS node users, it is possible to use `git` from SCL:
```

Comment:
What is `SCL`?

Reply:
Redhat Software Collection: https://developers.redhat.com/products/red-hat-software-collections/overview

Reply:
So RHSCL :)